src/js/redux/reducers/bulkDownload/bulkDownloadReducer.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * bulkDownloadReducer.js
+ * Created by Lizzie Salita 10/31/17
+ **/
+
+export const initialState = {
+    dataType: 'awards',
+    awards: {
+        award_levels: [],
+        filters: {
+            award_types: [],
+            agency: '',
+            sub_agency: '',
+            date_type: 'action_date',
+            date_range: {
+                start_date: '',
+                end_date: ''
+            }
+        },
+        columns: [],
+        file_format: 'csv'
+    },
+    agencies: [],
```

Comment:
What is the advantage to having `agencies` and `subagencies` in Redux rather than in the `AwardDataContainer`'s state?